.buildkite/pipelines/main/misc/sanitizers.yml

```diff
@@ -18,12 +18,14 @@ steps:
           gid: 1000
           workspaces:
             - "/cache/repos:/cache/repos"
-      # `contrib/check-asan.jl` needs a `julia` binary inside the inner sandbox:
+      # `contrib/asan/check.jl` needs a `julia` binary inside the inner sandbox:
       - JuliaCI/julia#v1:
           # Drop default "registries" directory, so it is not persisted from execution to execution
           persist_depot_dirs: packages,artifacts,compiled
           version: '1.6'
```

Comment:
Ah, thanks for catching this. Actually, since we don't run `check.jl` anymore since #42229, we don't need to install `julia` inside the sandbox. So, we can remove this step:

```
      - JuliaCI/julia#v1:
          # Drop default "registries" directory, so it is not persisted from execution to execution
          persist_depot_dirs: packages,artifacts,compiled
          version: '1.6'
```

Reply:
Good call. I've removed that step.